tests/test_pages.py

```diff
@@ -252,6 +243,8 @@ def allow_field(field_name):
         for prefix in ["_", "from_", "get_"]:
             if field_name.startswith(prefix):
                 return False
+        if field_name in {"cast"}:
+            return False
         return True
```

Comment:
```suggestion
        if field_name == "cast":
            return False
        return True
```

Reply:
I've added `{}` to handle a case if we add more methods in future, but it's probably pointless. Also, maybe all this dir-based code can be replaced with something from attrs.